examples/with-docker-compose/next-app/dev.Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
 RUN \
   if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
   elif [ -f package-lock.json ]; then npm ci; \
-  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i; \
+  elif [ -f pnpm-lock.yaml ]; then npm install -g pnpm && pnpm i; \
```

Comment:
```suggestion
  elif [ -f pnpm-lock.yaml ]; then corepack enable pnpm && pnpm i; \
```

Reply:
@styfle Is it `corepack enable pnpm` or just `corepack enable`?

Reply:
Depends if you want to enable all package managers or only pnpm

Reply:
I guess it makes sense as `corepack enable pnpm` in this case then!